index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Dire's Portfolio</title>
+    <script src="https://kit.fontawesome.com/b7a7cffe76.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;600&display=swap" rel="stylesheet" >
+</head>
+<body>
+  <main>
+    <header>
+      <div class="header">
+       <h1> <a href="">My Logo</a></h1>
+      <nav><a href=""><i class="fas fa-bars"></i></a></nav>
+      </div> 
+    </header>
+    <section class="container">
+      <div class="intro">
+        <header><h2>Hi! I'M DIRE</h2> 
```

Comment:
- [ ] Delete the header tag.
- [ ] Replace `h2` with a` h1`. This is important to add SEO support to your website.